packages/dids/package.json

```diff
@@ -76,41 +76,36 @@
   "dependencies": {
     "@decentralized-identity/ion-pow-sdk": "1.0.17",
     "@decentralized-identity/ion-sdk": "1.0.1",
+    "@noble/hashes": "^1.3.2",
+    "bittorrent-dht": "^11.0.5",
+    "bittorrent-dht-sodium": "^1.2.0",
```

Comment:
@decentralgabe any particular reason to list `bittorrent-dht-sodium` as an explicit dependency?  It doesn't appear to be used anywhere in the PR nor is it a dependency of any other packages as best I can tell.

Reply:
remnant, removing